srcpkgs/cgminer/template

```diff
@@ -12,10 +12,10 @@ makedepends="libcurl-devel libusb-devel jansson-devel eudev-libudev-devel ncurse
 conf_files="/etc/cgminer.conf"
 short_desc="Multi-threaded multi-pool CPU and GPU miner for bitcoin"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-license="GPL-3"
+license="GPL-3.0-or-later"
 homepage="https://github.com/ckolivas/cgminer"
 distfiles="http://ck.kolivas.org/apps/${pkgname}/${pkgname}-${version}.tar.bz2"
-checksum=debd49291f8335b48496896dc28a43f941d1c981525950497acc89d0d5fc1e84
+checksum=a8f3e244343eda8c3410e6d27188ee941de25c54bd76412e5b4d17de9f30ac18
 
 pre_configure() {
 	# We have latest jansson and libusb - just use them
```

Comment:
The following seds - why so much space between the sed and the options?

Reply:
Probably to line up to tabs.